### PR TITLE
[3572] Update new API parameter names

### PR DESCRIPTION
**Pull request by aachavarria**
https://github.com/craftercms/craftercms/issues/3572

### CODE REVIEW

File: ui/app/src/components/CreateSiteDialog.tsx [hunk 1]
```diff
@@ -520,12 +520,7 @@ function CreateSiteDialog() {
     };
     if (site.sandboxBranch) params.sandboxBranch = site.sandboxBranch;
     if (site.blueprintFields) params.siteParams = site.blueprintFields;
-    //TODO# remove this when change to Api2
```
Comment (rart) [inline ui/app/src/components/CreateSiteDialog.tsx:550]: What about API1 call which is still done for built in blueprints?

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/CreateSiteDialog.tsx (hunk 1 above)
Comment (aachavarria) [reply ui/app/src/components/CreateSiteDialog.tsx:550]: Built in blueprints are still using api1 create site

Comment (aachavarria) [reply ui/app/src/components/CreateSiteDialog.tsx:550]: built in blueprint uses fn "createParams" and marketplace uses fn "createMarketplaceParams"

**Review by rart — CHANGES_REQUESTED** [review]
Conflicts

**Comment by aachavarria** [comment]
conflicts fixed

**Review by rart — APPROVED** [review]

**Pull request merged**